install_config/persistent_storage/topics/glusterfs_static_provisioning.adoc

```diff
@@ -0,0 +1,179 @@
+. Create a GlusterFS volume. See the link:{gluster-admin-link}[{gluster} Administration Guide]
+for information on how to do this using the `gluster` command line interface or
+the link:https://github.com/heketi/heketi[heketi project site] for information
+on how to do this using `heketi-cli`. For this example, it will be called
+`myVol1`.
+
+. Define the following Service and Endpoints in `gluster-endpoints.yaml`:
```

Comment:
Do "Service" and "Endpoints" need to be capitalized? The Red Hat product glossary does not have them capitalized. If not, then this should be changed to "service" and "endpoints".

Reply:
Service and Endpoints in this case are Kubernetes/OpenShift resource types, much like StorageClass and PersistentVolume.